### PR TITLE
Closes #710: Add the number of opportunities on each stage

### LINKED ISSUE #710: I can see the number of opportunities on each stage
@Bonapara could you please provide a design for this? Thanks a lot

### PR DESCRIPTION
**Pull request by proof3**
Added number of opportunities on each stage column according to Figma design.

https://github.com/twentyhq/twenty/assets/27093731/b5b9bd0f-7d80-4b35-8281-e494e0423c4b

### CODE REVIEW

File: front/src/modules/ui/board/components/BoardColumn.tsx [hunk 1]
```diff
@@ -60,6 +60,27 @@ const StyledAmount = styled.div`
   margin-left: ${({ theme }) => theme.spacing(2)};
 `;
 
+const StyledNumChildren = styled.div`
+  align-items: center;
```
Comment (charlesBochet) [inline front/src/modules/ui/board/components/BoardColumn.tsx:64]: could you rewrite this CSS using the minimum of properties + using theme for all propreties (border-radius, font-size, gap, line-height, etc.) I don't think we need that much properties!

**Comment by charlesBochet** [comment]
Thanks for the help!

File: front/src/modules/ui/board/components/BoardColumn.tsx (hunk 1 above)
Comment (proof3) [reply front/src/modules/ui/board/components/BoardColumn.tsx]: I have removed the excess properties and have tried to use theme to describe all the properties. Let me know if I should change anything.
Thanks!!

File: front/src/modules/ui/board/components/BoardColumn.tsx [hunk 2]
```diff
@@ -60,6 +60,20 @@ const StyledAmount = styled.div`
   margin-left: ${({ theme }) => theme.spacing(2)};
 `;
 
+const StyledNumChildren = styled.div`
+  background-color: ${({ theme }) => theme.background.tertiary};
+  border-radius: ${({ theme }) => theme.border.radius.rounded};
+  color: ${({ theme }) => theme.font.color.tertiary};
+  line-height: ${({ theme }) => theme.text.lineHeight.lg};
+  margin-left: auto;
+  padding-bottom: ${({ theme }) => theme.spacing(1 / 2)};
```
Comment (charlesBochet) [inline front/src/modules/ui/board/components/BoardColumn.tsx:69]: let's actually use a height with justify-content: center + align-items: center actually.
using half padding seems hacky here :)

**Review by charlesBochet — APPROVED** [review]
Approved with a last comment :)

File: front/src/modules/ui/board/components/BoardColumn.tsx (hunk 2 above)
Comment (proof3) [reply front/src/modules/ui/board/components/BoardColumn.tsx:69]: Do you want me to remove the padding? Also adding height and justify-content: center + align-items doesn't seem to do anything. Maybe I've misunderstood what you mean.

Comment (proof3) [reply front/src/modules/ui/board/components/BoardColumn.tsx]: Done!

**Comment by charlesBochet** [comment]
That's perfect, thank you!
Merging :)

**Pull request merged**